packages/sanity/src/core/form/studio/tree-editing/components/breadcrumbs/TreeEditingBreadcrumbsTitle.tsx

```diff
@@ -0,0 +1,24 @@
+import {Box, Card, Flex, MenuDivider, Text} from '@sanity/ui'
+
+interface TreeEditingBreadcrumbsTitleProps {
+  title: string
+}
+
+export function TreeEditingBreadcrumbsTitle(props: TreeEditingBreadcrumbsTitleProps): JSX.Element {
+  const {title} = props
+
+  return (
+    <>
+      <Box paddingX={1}>
+        <Card padding={3}>
+          <Flex align="center" justify="space-between" gap={2}>
+            <Text size={1} muted textOverflow="ellipsis">
+              {title}
+            </Text>
+          </Flex>
+        </Card>
+      </Box>
+      <MenuDivider />
+    </>
```

Comment:
Nitpick: this could be a bit simplified to this (I don't think the `Flex` is necessary in this case).

```ts
<Card borderBottom padding={3} sizing="border">
  <Box paddingX={1} sizing="border">
    <Text size={1} muted textOverflow="ellipsis">
      {title}
    </Text>
  </Box>
</Card>
```